src/Service/ContentTypeService.php

```diff
@@ -4,9 +4,13 @@
 
 use Doctrine\Bundle\DoctrineBundle\Registry;
 use Doctrine\ORM\NoResultException;
+use Elastica\Aggregation\Terms;
```

Comment:
Don't like the use of the Elastica namespace in the coreBundle.

We have a search object in CommonBundle add a method, addTerms ?

Reply:
Fixed